packages/account/src/Components/file-uploader-container/file-uploader.jsx

```diff
@@ -119,6 +119,8 @@ const FileUploader = React.forwardRef(({ onFileDrop, getSocket }, ref) => {
     );
 });
 
+FileUploader.displayName = 'FileUploader';
```

Comment:
This line I had to add because of the error from **circleci** as 'Component definition is missing display name'. In other files the same situation.